pkgs/tools/security/onioncircuits/default.nix

```
@@ -1,31 +1,54 @@
-{ stdenv, lib, fetchgit, python3, intltool, gtk3, gobject-introspection, gnome }:
+{ stdenv
+, lib
+, fetchgit
+, gnome
+, gobject-introspection
+, gtk3
+, intltool
+, python3
+, wrapGAppsHook
+}:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "onioncircuits";
-  version = "0.5";
+  version = "0.7";
 
   src = fetchgit {
-    url = "https://git-tails.immerda.ch/onioncircuits/";
+    url = "https://gitlab.tails.boum.org/tails/onioncircuits";
     rev = version;
     sha256 = "13mqif9b9iajpkrl9ijspdnvy82kxhprxd5mw3njk68rcn4z2pcm";
   };
 
-  nativeBuildInputs = [ intltool ];
-  buildInputs = [ gtk3 gobject-introspection ];
-  propagatedBuildInputs =  with python3.pkgs; [ stem distutils_extra pygobject3 ];
+  format = "other";
 
-  postFixup = ''
-    wrapProgram "$out/bin/onioncircuits" \
-      --prefix GI_TYPELIB_PATH : "$GI_TYPELIB_PATH" \
-      --prefix XDG_DATA_DIRS : "$out/share:${gnome.adwaita-icon-theme}/share"
+  nativeBuildInputs = [ wrapGAppsHook intltool gobject-introspection ];
+  propagatedBuildInputs = with python3.pkgs; [ stem distutils_extra pygobject3 ];
```

```suggestion
  nativeBuildInputs = [
    wrapGAppsHook
    intltool
    gobject-introspection
  ];

  propagatedBuildInputs = with python3.pkgs; [
    stem
    distutils_extra
    pygobject3
  ];
```

Increases readability, maybe we can also sort those lists.